### PR TITLE
add new column toggle

### PR DESCRIPTION
**Pull request by jamzhong15**
in progress

### CODE REVIEW

File: src/components/staffing/StaffingTable.js [hunk 1]
```diff
@@ -26,6 +29,9 @@ export default class StaffingTable extends React.Component {
         maxHeight={750}
         headerHeight={40}
       >
+        
+        {
+        (this.props.columnOrder.includes('User') || this.props.columnOrder.includes('all')) && (
```
Comment (SimonLTheodo) [inline src/components/staffing/StaffingTable.js:34]: Is there any way to make this a bit more readable? Maybe put the || part in its own variable

File: src/components/staffing/StaffingTable.js [hunk 2]
```diff
@@ -38,6 +44,13 @@ export default class StaffingTable extends React.Component {
           width={200}
           fixed
         />
+        )
+
+        }
+        
+
+        {
+        (this.props.columnOrder.includes('Company') || this.props.columnOrder.includes('all')) && (
```
Comment (SimonLTheodo) [inline src/components/staffing/StaffingTable.js:53]: Same as above

File: src/components/staffing/StaffingTable.js (hunk 1 above)
Comment (jamzhong15) [reply src/components/staffing/StaffingTable.js:34]: true thanks

Comment (jamzhong15) [reply src/components/staffing/StaffingTable.js:34]: didnt seem to work, will make a future ticket for that

**Review by SimonLTheodo — APPROVED** [review]

**Pull request merged**